server/py/services/api/api/endpoints/projects.py

```diff
@@ -464,7 +464,7 @@ async def load_project(
     )
 
     # We must create the project before we run the remote load_project function because
-    # we want this function will be running under the project itself instead of the default project.
+    # we want this function will be running under the project itself instead of the active project.
```

Comment:
This comment is not relevant anymore because neither the default and active project are relevant in the server now.